CMake/run_rmg.cmake

```diff
@@ -40,7 +40,7 @@ else(QMC_NO_SLOW_CUSTOM_TESTING_COMMANDS)
   endfunction()
 
   function(ADD_RMG_CONVERT_TEST TESTNAME PREFIX WORKDIR TEST_INPUT)
-    add_test(NAME ${TESTNAME} COMMAND $<TARGET_FILE:convert4qmc> -rmg ${TEST_INPUT} -prefix ${PREFIX})
+    add_test(NAME ${TESTNAME} COMMAND $<TARGET_FILE:convert4qmc> -nojastrow -rmg ${TEST_INPUT} -prefix ${PREFIX})
     set_tests_properties(${TESTNAME} PROPERTIES WORKING_DIRECTORY ${WORKDIR})
     set_property(TEST ${TESTNAME} APPEND PROPERTY LABELS "converter;rmg")
```

Comment:
The issue is that test `${FULL_NAME}-rmg2qmc` only has dependency on `${FULL_NAME}-scf` at line 60.
However it should also depend on `LINK_${SOURCE}_h5_Waves` `LINK_${SOURCE}_TO_${TARGET}` and `COPY_${SOURCE}_XML_TO_${TARGET}`

Reply:
I think the  `${FULL_NAME}-rmg2qmc` may only need `LINK_${SOURCE}_h5_Waves` (the `rmg2qmc` only calls convert4qmc, and `LINK_${SOURCE}_h5_Waves` already depends on `${SOURCE}-scf`).
The `run_rmg_test` (whatever `${FULL_NAME}` is on line 60) will need the xml output, so maybe I should add `set_tests_properties(${FULL_NAME} PROPERTIES DEPENDS ${FULL_NAME}-rmg2qmc)` after line 60.

Reply:
I cannot find a test named `${FULL_NAME}`